mindmeld/components/nlp.py

```diff
@@ -340,6 +340,11 @@ def _process_list(self, items, func, *args, **kwargs):
     def _validate_locale(self, locale=None):
         """This function makes sure the locale is consistent with the app's language code"""
         locale = locale or self.locale
+
+        # if the developer or app doesnt specify the locale, we just use the default locale
```

Comment:
```suggestion
        # if the developer or app doesn't specify the locale, we just use the default locale
```

Reply:
what is the default locale in this context? the default locale of the language?

Reply:
Whatever Duckling uses when the locale passed-in in None